crates/binstalk-downloader/src/remote/resolver.rs

```diff
@@ -3,7 +3,7 @@ use std::{net::SocketAddr, sync::Arc};
 use hyper::client::connect::dns::Name;
 use once_cell::sync::OnceCell;
 use reqwest::dns::{Addrs, Resolve};
-use tracing::{info, instrument, warn};
+use tracing::{instrument, warn};
```

Comment:
Missing `debug` import:

```suggestion
use tracing::{debug, instrument, warn};
```

Reply:
You also need to remove the `debug` in `cfg(windows)`, however I cannot comment on unchanged line